### DISCUSSION TITLE
Support for ppc64le and s390x

### DISCUSSION THREAD

**Discussion by srbala** [firecracker-microvm/firecracker#3049] [Q&A] [answered]
Is firecracker can be compiled and used in ppc64le and s390x arch environments?

**Comment by dianpopa** [comment, marked as answer]
Hi,

Firecracker does not have support for PowerPC and s390x archs. You can read more on the supported platforms here: https://github.com/firecracker-microvm/firecracker/blob/main/README.md#supported-platforms.
